app/src/main/java/org/p2p/wallet/send/repository/SendStorage.kt

```diff
@@ -0,0 +1,30 @@
+package org.p2p.wallet.send.repository
+
+import androidx.core.content.edit
+import android.content.SharedPreferences
+import org.p2p.core.crypto.Base58String
+import org.p2p.core.crypto.toBase58Instance
+
+class SendStorage(
+    private val prefs: SharedPreferences
+) : SendStorageContract {
+    private companion object {
+        const val KEY_TOKEN_MINT = "TOKEN_MINT"
+    }
+
+    override fun restoreFeePayerToken(): Base58String? {
```

Comment:
restore is more like domain layer method
storage only set/save get/require

Reply:
yep, refactored this point a bit